### PR TITLE
Podcast page

### PR DESCRIPTION
**Pull request by danielstreif**
**What**:
- added a podcast site with embedded youtube videos of Die Lernwerkstatt
- created a form to add further videos
- added and updated i18n translations

**Why**:
- Die Lernwerkstatt started a podcast/video series and wanted to access the videos through their site

**How**:
- the structure and forms of the blog site were reused and adjusted for the videos
- videos are embedded via iframe, the no-cookie setting is used to prevent youtube from collecting data
- data is saved in an Atlas MongoDB collection
- the title "Podcast" and the layout as separate page were discussed and agreed upon with Die Lernwerkstatt

Podcast site:
image

Podcast form:
image

**Checklist**:

- [x] Tests
- [x] Ready to be merged

### CODE REVIEW

File: src/views/podcastform.hbs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{{#section "head"}}
+<title>Podcasts - Die Lernwerkstatt</title>
```
Comment (jarjan) [inline src/views/podcastform.hbs:2]: i18n this title too :)

**Review by jarjan — APPROVED** [review]

File: src/views/podcasts.hbs [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+{{#section "title"}}
+<div class="container my-4">
+  <div class="row">
+    <div class="col border-bottom">
+      <h1 class="h1">{{{__ "podcasts_title"}}}</h1>
+    </div>
+  </div>
+</div>
+{{/section}}
```
Comment (pheeria) [inline src/views/podcasts.hbs:13]: I thought we had a template for that 🤔

File: src/views/partials/podcasts/create.hbs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <input type="submit" value="{{{__ "common_submit"}}}">
+    <input type="reset">
+  </div>
+</form>
+
+<script>
+  var simplemde = new SimpleMDE({
+    element: document.getElementById("main-content")
+  });
+</script>
```
Comment (pheeria) [inline src/views/partials/podcasts/create.hbs:27]: Do we need this or was it copy-pasted from the blog and forgotten?

File: src/views/podcasts.hbs (hunk 2 above)
Comment (danielstreif) [reply src/views/podcasts.hbs:13]: You are right...had to search for it. For some reason we are not using it though. Seems to me like it doesn't work well with our i18n options. We only use it in places where there are no translations (yet). The team page is an example:
`{{> main/title title="Wir sind Die Lernwerkstatt"}}`

File: src/views/partials/podcasts/create.hbs (hunk 3 above)
Comment (danielstreif) [reply src/views/partials/podcasts/create.hbs:27]: Forgotten 😅

File: src/views/podcasts.hbs (hunk 2 above)
Comment (pheeria) [reply src/views/podcasts.hbs:13]: @jarjan any inputs?

File: src/views/partials/podcasts/create.hbs [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<link rel="stylesheet" href="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.css">
+<script src="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.js"></script>
```
Comment (pheeria) [inline src/views/partials/podcasts/create.hbs:2]: These as well, no? 🙂

File: src/views/podcasts.hbs (hunk 2 above)
Comment (jarjan) [reply src/views/podcasts.hbs:13]: Perhaps I created this component, but didn't refactored eveywhere.

File: src/views/partials/podcasts/create.hbs (hunk 4 above)
Comment (danielstreif) [reply src/views/partials/podcasts/create.hbs]: True, I left the content section in at first to add descriptions to the videos. I decided against it in the end but must have forgotten to double check the rest of the file. Thanks for spotting it 👍

**Review by pheeria — APPROVED** [review]
Thanks 🎉

**Pull request merged**